src/main/java/org/spongepowered/api/Server.java

```diff
@@ -206,6 +206,49 @@
     Optional<WorldProperties> createWorld(WorldCreationSettings settings);
 
     /**
+     * Creates a copy of the provided world under the new name given and returns
+     * the new copy if the copy was possible. 
+     *
+     * @param originalWorld The original world instance to copy
+     * @param copyName The name that should be used in the duplicated entry instead of the originalWorld's name
+     * @return An {@link Optional} containing the properties of the new world instance, if the copy was successful
+     */
+    Optional<WorldProperties> copyWorld(WorldProperties originalWorld, String copyName);
+
+    /**
+     * Allows the copy to be made while the world is still loaded by forcing a save
+     * first, this could cause changes to be missed if made after that save. By
+     * using this method the developer is assumed to properly respect that large worlds
+     * will likely take a long enough period to copy that a reasonable number of changes are
+     * missed.
+     *
+     * A {@link World} instance is used here because unloaded worlds should not be copied
+     * using this method, and instead should be copied using {@link #copyWorld(WorldProperties,String)}.
+     *
+     * @param originalWorld The original world instance to copy
+     * @param copyName The name for the copy that is made
+     * @return An {@link Optional} containing the properties of the new world instance, if the copy was successful
+     */
+    Optional<WorldProperties> copyLoadedWorld(World originalWorld, String copyName);
```

Comment:
Why have a second method for this? Loaded worlds can be handled silently in implementation. Just state in copyWorld if the world being copied is already loaded the following will happen

1. A force save will occur
2. world saving, for the world being copied, is temporarily disabled
3. Copy starts
4. After copy is complete, world saving is re-enabled.